Index: xmlcopyeditor-1.2.0.12/src/xmlcopyeditor.cpp
===================================================================
--- xmlcopyeditor-1.2.0.12.orig/src/xmlcopyeditor.cpp	2014-01-29 20:31:03.830888398 +0100
+++ xmlcopyeditor-1.2.0.12/src/xmlcopyeditor.cpp	2014-01-29 20:31:03.826888394 +0100
@@ -1198,8 +1198,7 @@
 					}
 					else
 					{
-						std::cout << "Usage: xmlcopyeditor [--version --help -ws] [<file>] [<file2>]" << std::endl
-						    << "Options -w (import Microsoft Word document) and -s (open Spelling and style check) are provided for integration with Microsoft Office and only available on Windows" << std::endl;
+						std::cout << "Usage: xmlcopyeditor [--version --help] [<file>] [<file2>]" << std::endl;
 					}
 					exit ( 0 );
 				default:
